src/Okta.Sdk.IntegrationTests/FactorScenarios.cs

```diff
@@ -58,6 +58,48 @@ public async Task CreateSecurityQuestionFactor()
             }
         }
 
+        [Fact]
+        public async Task EnrollSmsFactor()
+        {
+            var client = TestClient.Create();
+            var guid = Guid.NewGuid();
+
+            var profile = new UserProfile
+            {
+                FirstName = "Jill",
+                LastName = "Factor-Sms",
+                Email = $"jill-factor-sms-dotnet-sdk-{guid}@example.com",
+                Login = $"jill-factor-sms-dotnet-sdk-{guid}@example.com",
+            };
+            profile["nickName"] = "jill-factor-sms";
+
+            var createdUser = await client.Users.CreateUserAsync(new CreateUserWithPasswordOptions
+            {
+                Profile = profile,
+                Password = "Abcd1234",
+            });
+
+            try
+            {
+                await createdUser.AddFactorAsync(new AddSmsFactorOptions()
+                {
+                    PhoneNumber = "+15554151337",
```

Comment:
What phone number is this? 
For node and java SDK, we use a google voice number - https://github.com/okta/okta-sdk-nodejs/blob/master/test/it/factor-create.js#L87

Reply:
I grabbed this number from [docs](https://developer.okta.com/docs/api/resources/factors/#enroll-okta-sms-factor). We don't need a real number for this specific test. Besides the consistency, is there any other reason to use that?